chef-dk/plan.sh

```diff
@@ -0,0 +1,145 @@
+pkg_name=chef-dk
+pkg_origin=core
+pkg_maintainer="The Chef Maintainers <humans@chef.io>"
+pkg_description="The Chef Developer Kit"
+pkg_license=('Apache-2.0')
+pkg_bin_dirs=(bin)
+pkg_build_deps=(
+  core/make
+  core/gcc
+  core/coreutils
+  core/git
+)
+
+pkg_deps=(
+  core/glibc
+  core/busybox-static
```

Comment:
There is a RFC for this that is being discussed ( #1516 ), but it would be best if we could use other packages instead of `busybox` here.